docs/proposals/Drawer/README.md

```diff
@@ -0,0 +1,70 @@
+# Drawer
+
+The `<Drawer>` component will provide a dismissible side panel.
+
+## Design Patterns
+
+`<Drawer>` It will allow the user to view related content to their main task.
+Unlike a `<Modal>`, it will take full advantage of the available viewport
+height. This will make it ideal for long lists and other scrolling content.
```

Comment:
Other than the placement is there anything semantically different between a drawer and a modal?

Do you see users being able to open a drawer and then be able to continue interacting with the base page?

Reply:
Yes, it would be excellent to have an "inline" version that displaces the content rather than overlays it. We're using the work overview one in our prototype dashboard right now.

![image](https://user-images.githubusercontent.com/1479091/74574766-ef087980-4f41-11ea-8f2e-56ce1ad26981.png)

Reply:
Hmmm, I was thinking that the activity feed could be changed from a modal to a drawer with an overlay but the behaviour would be similar. The biggest difference that I could see would be a sliding animation... which I haven't really thought about yet.

Reply:
A close analogue would be the 'temporary' and 'persistent' drawers in Material Design: https://material-ui.com/components/drawers/.